app/controllers/application_controller.rb

```diff
@@ -269,13 +269,17 @@ def access_denied(options ={})
   def admin_only_access_denied
     respond_to do |format|
       format.html do
-        flash[:error] = ts("Sorry, only an authorized admin can access the page you were trying to reach.")
+        flash[:error] = t("admin.access.page_access_denied") 
         redirect_to root_path
       end
       format.json do
-        errors = [ts("Sorry, only an authorized admin can do that.")]
+        errors = [t("admin.access.action_access_denied")]
         render json: { errors: errors }, status: :forbidden
       end
+      format.js do
+        flash[:error] = t("admin.access.page_access_denied") 
+        render js: "window.location.href = '#{root_path}';" 
+      end
```

Comment:
Might be a silly question, but to make sure I understand, what requires adding this format?

Reply:
This is for redirecting user and showing the error when handling an ajax request. In this case, when the admin tries to approve an inbox comment, they will be redirected to the root page with the error flashing above it. I tried to show the error on the same inbox page, but I wasn't able to do that. If there's a better way of doing this, please let me know